File: matrices/matricesSum.java
package springIntro;

public class matricesSum {

	public static void main(String[] args) {
		int matrix1[][] = {{3, 5, 9}, {4, 6, 1}, {1, 8, 2}};
		int matrix2[][] = {{1, 0, 3}, {4, 5, 7}, {2, 3, 6}};
		int i = 0, j = 0;
		while(i < 3) {
			while(j < 3) {
				System.out.printf("%d ", matrix1[i][j] + matrix2[i][j]);
				j++;
			}
			System.out.printf("\n");
			j = 0;
			i++;
		}
	}

}

File: matrices/matricesSum.cpp
#include<iostream>
using namespace std;
int main(){
	int matrix1[3][3] = {{3, 5, 9}, {4, 6, 1}, {1, 8, 2}};
	int matrix2[3][3] = {{1, 0, 3}, {4, 5, 7}, {2, 3, 6}};
	for(int i = 0; i < 3; i++){
		for(int j = 0; j < 3; j++){
			cout << matrix1[i][j] + matrix2[i][j] << " ";
		}
		cout << endl;
	}
	return 0;
}
